app/views/users/profile.html.erb

```diff
@@ -70,8 +70,8 @@
 
     <div class="tab-pane" id="questions">
       <div class="btn-group text-center">
-        <a href="#asked" class="btn btn-default active" data-toggle="tab">Asked</a>
-        <a href="#answered" class="btn btn-default" data-toggle="tab">Answered</a>
+        <a href="#asked" class="btn btn-default" data-toggle="tab">Asked<%= "(#{Node.questions.where(status: 1, uid: @profile_user.id).length})" %></a>
```

Comment:
Should we use `count` here, so it counts the records in the db, and doesn't have to fetch them all and then count them? Thank you!!!